.changeset/fuzzy-dancers-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Websocket support in `dev`
```

Comment:
It would be great if we could get a bit more content into these changesets. 🙏

Reply:
Done!